src/components/CallView/shared/VideoVue.vue

```diff
@@ -680,24 +689,35 @@ export default {
 	top: calc(50% + 80px);
 }
 
-.speaking-shadow {
+.video-container::after {
+	content: '';
```

Comment:
```suggestion
```
That will prevent ::after pseudo element from rendering, unless it's needed. You still have this rule for subclasses, so should be fine